management/server/grpcserver.go

```diff
@@ -359,9 +367,23 @@ func (s *GRPCServer) Login(ctx context.Context, req *proto.EncryptedMessage) (*p
 		}
 	}
 
+	// check if peer login has expired
```

Comment:
The feeling to write a inline comment is a good indicator to should write a new function whit a proper name.

Reply:
makes sense